.copyrightignore

```diff
@@ -20,31 +20,34 @@
 #
 # ===========================================================================
 #
-#  THE FOLLOWING ARE EXAMPLES OF SYNTAXES copyrightCheck SUPPORTS:
-#  1.**/jobs/** # path contains "jobs" folder, but not root directory
-#  2./doc#root root level directory and file named "doc"
-#  3./debugtools/**/*.ignore # all .ignore file in the root directory debugtools
-#  4./*.ignore #all root directory .ignore files
-#  5.**/jobs/pipelines/ #pipeline folder follows closely with jobs not in root directory
-#  6.*.ignore #all ignore files only
-#  7.**/jobs/**/pipelines/ #directory path contains both jobs and pipelines not root directory
-#  8.*job*  #all things at contains characters in between stars
+# THE FOLLOWING ARE EXAMPLES OF SYNTAXES copyrightCheck SUPPORTS:
+# 1. **/jobs/** # path contains "jobs" folder, but not root directory
+# 2. /doc # root level directory and file named "doc"
+# 3. /debugtools/**/*.ignore # all .ignore files in the root directory debugtools
+# 4. /*.ignore # all root directory .ignore files
+# 5. **/jobs/pipelines/ # pipeline folder follows closely with jobs not in root directory
+# 6. *.ignore # all ignore files only
+# 7. **/jobs/**/pipelines/ # directory path contains both jobs and pipelines not root directory
+# 8. *job* # all things at contains characters in between stars
 # ===========================================================================
 
 src/bsd/doc
 src/jdk.crypto.ec/share/native/libsunec/impl
 src/linux/doc
+src/share/lib/security/java.policy
+src/share/lib/security/java.security-*
```

Comment:
Do we know why these aren't prepended with `jdk/` part of the path?

Reply:
I followed the existing pattern: paths like this should be ignored whether they're within `jdk` or within `closed/overrides/jdk`.